opentreemap/treemap/models.py

```diff
@@ -1230,6 +1230,7 @@ class Boundary(models.Model):
                                       db_index=True)
 
     canopy_percent = models.FloatField(null=True)
+    searchable = models.BooleanField(default=True)
```

Comment:
This should populate existing rows with a default value for backwards compatibility. Do you prefer making this a `NullBooleanField` instead?